packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -37,16 +37,18 @@ @interface FLTImagePickerPlugin () <UINavigationControllerDelegate,
 
 @property(strong, nonatomic) PHPickerViewController *pickerViewController API_AVAILABLE(ios(14));
 
+@property(strong, nonatomic) UIImagePickerController *imagePickerController;
```

Comment:
Optional nit, since it's a pre-existing problem: Could you add a comment for this property? Properties and ivars should, by style, have declaration comments.

Reply:
Added the declaration comments as required by style.